ex2.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="ex1_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="ex2_header_css.css">
+	<link rel="stylesheet" type="text/css" href="buttons.css">
+
+</head>
+<body>
+
+	<section>
+		<header class="header"> 
+			<img src="logo.jpg" style="float:left; width:200px;height:40px; margin-left: 3%">
+			<h2><a href="ex2.html" target="_self" class="button1" style="margin-left: 7%; margin-right: 0.1%;">Evaluations</a></h2>
```

Comment:
Can you explain why you chose to wrap these anchor tags in heading levels? Furthermore, could you clarify why you chose different heading levels?